onprc_ehr/resources/queries/study/PMIC_ChargeTypes.sql

```diff
@@ -1,3 +1,3 @@
 SELECT chargetype
-From Site.{substitutePath moduleProperty('onprc_billing','BillingContainer')}.onprc_billing.chargeUnits
+From Site.{substitutePath moduleProperty('ONPRC_Billing','BillingContainer_Public'}.onprc_billing.chargeUnits
```

Comment:
Our automated tests pointed out a typo here - the missing closing parens

```suggestion
From Site.{substitutePath moduleProperty('ONPRC_Billing','BillingContainer_Public')}.onprc_billing.chargeUnits
```